src/main/java/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package db;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class Storage {
+    public static final List<FruitTransaction> transactions = new ArrayList<>();
```

Comment:
```suggestion
    public static final Map<String, Integer> fruitStorage = new ArrayList<>();
```
тобі потрібно зберігати фрукти і їх кількість